modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -6821,6 +6820,7 @@ export-overridden-translations=Export Overridden Translations
 export-permissions=Export Permissions
 export-personal-data=Export Personal Data
 export-private-pages=Export Private Pages
+export-process-deletion-warning-message=Export process and export LAR file will be deleted after 30 days.
```

Comment:
It should be committed under the commit `LPS-166513 adding alert message for export expiry`

Reply:
@tototrinh  it is in the commit `LPS-166513 adding alert message for export expiry`. it just appear in the buildlang too because when running `blade gw buildLang` the file was reformatted and its position change to adhere to the alphabetic order of the keys please check the [commit](https://github.com/tototrinh/liferay-portal/pull/55/commits/4625eafce900b9617b9a6c6e1b9af42957631f30)

Reply:
Hi @thabet91, `blade gw buildLang` also help SF but any changes in `language. properties` should not be included in `buildLang` commit. Thanks.

Reply:
@tototrinh fixed